src/scripts/main.js

```diff
@@ -1,11 +1,24 @@
 'use strict';
 
+const body = document.querySelector('body');
+
 function waitFor(element, eventName) {
-// write your code here
+  const promise = new Promise((resolve, reject) => {
+    element.addEventListener(`${eventName}`, () => {
```

Comment:
```suggestion
    element.addEventListener(eventName, () => {
```
eventName already string